api/db.ts

```diff
@@ -0,0 +1,4 @@
+//perhaps we want just regular postgres library, but pg-promise will be fine
```

Comment:
I do agree, I'm looking into if we loose any functionality by using the pg library. (It uses promises as well I just need to check with my desktop to verify.)